initial Fastlane structures from IzzyOnDroid

IzzySoft opened the pull request:
this PR provides you with a "fastlane starter package" (as announced in #6). Some notes to sum them up in a place easy to find for you:

* the [IzzyOnDroid Fastlane Documentation](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Fastlane) can guide you with maintaining the structures here
* take special care for limits, e.g. per-release changelogs can have max 500 chars, `full_description.txt` max 4,000 chars, graphics must have specific aspect ratios etc.
* you can have "bigger graphics" here; the IoD updater takes care to resize/optimize them as needed
* for the `full_description.txt` here I've used HTML compressed into a single line, to prevent the fdroid software converting each line break to a `<br>`. The tags used for this are supported by F-Droid.org as well, and to my knowledge even by PlayStore.
* Details on all this in the linked documentation.
* Should you have questions, be welcome to ask :wink:

And now: Enjoy!

IzzySoft commented:
Oof. You#re unstoppable fast, thanks!

Btw, talking about "project description":

> This version has been "abandoned" in favour of tag based v3, if you don't like tags you can still use this, but it will not receive major updates

Seeing how swift you act and made new releases, is that some "forgotten hint"? Reads a bit discouraging, especially being continued by

> v3 has no current release schedule and might be released as ArruTag

Could scare some away, getting the impression of "oh, this is no longer maintained – and the successor far from being ready". Can you maybe outline how it's meant? Might suggest a different phrasing then – unless above interpretation is correct of course.

KSSidll commented:
Yeah, good point

Tbf that whole statement is kinda nonsensical, I just got a bit scared because of the unexpected 'attention'

App was developed by me mainly for me so as I use it daily it will be maintained until I bang my head into the tag system for long enough to figure out how to do it properly and it being as user-friendly as possible, which will take some time due to my limited time lately

I got rid of the whole thing

IzzySoft commented:
Oy… Scaring you was not the intention here at all!

And yeah, many of the FOSS apps started as personal projects. Most of my own were. Even IzzyOnDroid was when it started, about 14 years ago: I switched to Android and was looking what apps to use as replacements for my previous Symbian ones, so I made myself lists (in a Forum back then). Those got ever longer, and plenty of positive feedback came in – so I started my own website with them. Had the lists self-updating. People came asking things, so I started my blog on the same site. Then I dropped out of the Google ecosystem, degoogling my devices as I didn't want their surveillance, tracking etc. So where to get the apps now? Especially those missing at F-Droid? So the IzzyOnDroid repo was born.

I could continue that for even more – not just on Android, but also e.g. my eBook server…

> which will take a while due to my limited time lately

No rush! It's your hobby. It goes at your pace.

> I got rid of the whole thing

Nice! And wishing you all the best for your project(s). Should you need us from IzzyOnDroid, be welcome to give us a ping. We try to help when we can :smiley:
